repository-structure/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- ====================================================================== -->
+    <!-- P A R E N T  P R O J E C T  D E S C R I P T I O N                      -->
+    <!-- ====================================================================== -->
+    <parent>
+        <groupId>com.hcl.ecomm</groupId>
+        <artifactId>hclecomm</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <!-- ====================================================================== -->
+    <!-- P R O J E C T  D E S C R I P T I O N                                   -->
+    <!-- ====================================================================== -->
+    <artifactId>hclecomm.repository-structure</artifactId>
+    <packaging>content-package</packaging>
+    <name>hclecomm - Repository Structure Package</name>
+    <description>
+        Empty package that defines the structure of the Adobe Experience Manager repository the Code packages in this project deploy into.
+        Any roots in the Code packages of this project should have their parent enumerated in the Filters list below.
+    </description>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <properties>
+                        <cloudManagerTarget>none</cloudManagerTarget>>
+                    </properties>
+                    <filters>
+
+                        <!-- /apps root -->
+                        <filter><root>/apps</root></filter>
+
+                        <!-- Common overlay roots -->
+                        <filter><root>/apps/sling</root></filter>
+                        <filter><root>/apps/cq</root></filter>
+                        <filter><root>/apps/dam</root></filter>
+                        <filter><root>/apps/wcm</root></filter>
+                        <filter><root>/apps/msm</root></filter>
+
+                        <!-- Immutable context-aware configurations -->
+                        <filter><root>/apps/settings</root></filter>
+						
+						<!-- /etc root -->
+						<filter><root>/etc</root></filter>
+
+                    </filters>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```

Comment:
@rakeshkumar-verma  getting build error while compiling the code for apps.
![image](https://user-images.githubusercontent.com/63444465/86344809-5ff1c900-bc78-11ea-926e-5cc75306ead9.png)